main.go

```diff
@@ -37,13 +45,20 @@ const (
 
 var vetPurl = "pkg:golang/safedep/vet@" + version
 
-const banner string = `
-Yb    dP 888888 888888
- Yb  dP  88__     88
-  YbdP   88""     88
-   YP    888888   88
+func generateVetBanner(version, commit string) string {
```

Comment:
Let's move the banner generation to DRY for consistency? Define the colours as constants in DRY. 

The tool can invoke PrintBanner with a config? We can put this in ui/cli package in DRY. 

WDYT?